docs/specs/tripgo.swagger.yaml

```diff
@@ -835,7 +835,7 @@ paths:
         required: true
         description: Identifier of trip as returned by previous server call.
 
-  /trip/{id}/save:
+  /trip/save/{id}:
```

Comment:
Out of curiousity, what's the reason for this change? I think the old style was more RESTy.

Reply:
This is just to be consistent with our endpoints. Agree that the other way is better, but will create a PR and then update the docs.